python/action-server/main.py

```diff
@@ -0,0 +1,154 @@
+import time
+import json
+import itertools
+import zenoh
+from zenoh import Reliability, SampleKind, Query, Sample, KeyExpr, QueryTarget, Value
+from pydantic import BaseModel
+import logging
+import asyncio
+import yaml
+import state_machine
+log = logging.getLogger(__name__)
+
+# describe settings for parsing values.
+class ActionSettings(BaseModel):
+    mode: str = None
```

Comment:
`mode: str = “”` is type wise correct.

otherwise this would be 
`mode: Optional[str] = None` - not sure why this approach would be useful

Reply:
Updated code with first approach.